tests/cpp/test_allocation_order_inference.cpp

```diff
@@ -25,6 +25,16 @@ using testing::ElementsAre;
 
 using AllocationOrderInferenceTest = NVFuserTest;
 
+std::vector<int64_t> getAllocationDomainPermutation(TensorView* tv) {
+  std::optional<std::vector<int64_t>> permutation =
+      ir_utils::computePermutation(
+          tv->getMaybeRFactorDomain(), tv->getMaybeAllocationDomain());
+  if (permutation.has_value()) {
+    return permutation.value();
+  }
+  return {};
```

Comment:
Wdyt about
```suggestion
  return *permutation;
```
for YAGNI and less ambiguity on what an empty order means. As is, it means either not a permutation or a valid permutation on a rank-0 tensor.